tests/acceptance/image-sets-test.js

```diff
@@ -32,3 +33,35 @@ test('visiting /image-sets', function() {
     equal(currentURL(), '/image-set/24');
   });
 });
+
+test('delete /image-set/:id', function() {
+  expect(5);
+
+  stubRequest('delete', '/imageSets/24', function(request){
+    ok(true, 'delete api was called');
+    return this.noContent(204);
```

Comment:
@redyaffle I fixed that test. A couple of things to note
 1. I moved the stub to inside the test instead of in the fake-requests file. I think it's useful for this type of test to assert that the api was actually hit, which we do with the `ok` and then, importantly, the `expect(5)` above. If you comment out the ok(true) line, then the expect will fail. This is similar to [this test or making sure the API is hit with correct search params](https://github.com/201-created/LG/blob/master/tests/acceptance/lions/search-test.js#L47-L56)
2. You have to call this.noContent instead of this.success. The reason for the error you saw is that when ember calls the API /imageSets, even with a delete request, if it gets any content back, it expects that to be JSON and tries to serialize the JSON instead an imageSet, so it's expecting an ID parameter.

Let me know if you have any other questions. I think this is ready to merge.